gui/pages/disk_config.go

```diff
@@ -784,18 +784,22 @@ func (disk *DiskConfig) populateComboBoxes() error {
 	}
 
 	if disk.isSafeSelected {
+		selected := 0
 		if len(disk.safeTargets) > 0 {
 			disk.chooserCombo.SetSensitive(true)
-			for _, target := range disk.safeTargets {
+			for n, target := range disk.safeTargets {
 				log.Debug("Adding safe install target %s", target.Name)
 				err := addListStoreMediaRow(safeStore, target)
 				if err != nil {
 					log.Warning("SetValue safeStore")
 					return err
 				}
+				if target.Name == disk.model.InstallSelected[target.Name].Name {
+					selected = n
+				}
 			}
 			disk.chooserCombo.SetModel(safeStore)
-			disk.chooserCombo.SetActive(0)
+			disk.chooserCombo.SetActive(selected)
```

Comment:
Sometimes when I switch between `Safe Install` and `Destructive Install` my selection is not remembered

Reply:
will be done in a separate PR